### ISSUE TITLE
latex.now.sh unreachable

### ISSUE THREAD

**Issue by Starbix** [vincentdoerig/latex-css#69]
It seems like now.sh is now unreachable. latex.vercel.app still works, so it seems vercel just now fully removed support for now.sh.

**Comment by vincentdoerig** [comment]
Thank you for the notice! I have opened a [question](https://github.com/orgs/vercel/discussions/7300) on Vercel's community forum to request further clarification from them. In addition, I have pushed some changes (ee2bb85), renaming all references of the now.sh domains to vercel.app. I hope that the impact of this failed redirection is minimal and that Vercel might consider restoring the redirection soon…
